README.md

```diff
@@ -18,7 +18,7 @@ The Pipeline Model Definition Plugin provides a config-like syntax for defining
 - To start developing plugin: 
 - Clone the plugin repository
 
-`cd pipeline-model-definition && mvn install && mvn hpi:run`
+`cd pipeline-model-definition-plugin && mvn install && mvn hpi:run`
```

Comment:
@dwnusbaum already spotted this, but running this way won't provide Declarative's functionality. You'll just end up with a pretty vanilla Jenkins that can only create a Freestyle job:

<img width="1377" alt="image" src="https://user-images.githubusercontent.com/21689198/83191147-8ebcd280-a101-11ea-99f0-b4ba2c4aaf29.png">

`mvn -f pipeline-model-definition/pom.xml hpi:run` can work, or you can `cd pipeline-model-definition` then `mvn hpi:run` that way.

My suggestion: do away with the change to the README. I think this could be made more clear, but maybe it's for a different PR.